### PR TITLE
remove [final_radiological_review]

### PR DESCRIPTION
**Pull request by kongtiaowang**
This pull request removes [final_radiological_review] module and all files which relate to this module.

Adding a release patch for SQL.

### CODE REVIEW

**Comment by driusan** [comment]
This should probably go to the major branch

File: modules/conflict_resolver/php/conflict_resolver.class.inc [hunk 1]
```diff
@@ -11,7 +11,7 @@
  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  * @link     https://github.com/aces/Loris
  */
-
+namespace LORIS\conflict_resolver;
```
Comment (driusan) [inline modules/conflict_resolver/php/conflict_resolver.class.inc]: I think there's something wrong with this PR. This (and a bunch of files below) are unrelated and seem to be from something else..

Comment (kongtiaowang) [reply modules/conflict_resolver/php/conflict_resolver.class.inc]: @driusan 
I messed up in this branch.
I will close this and send a new one.

**Comment by kongtiaowang** [comment]
The new https://github.com/aces/Loris/pull/3250 will replace this one.

**Pull request closed without merging**